### ISSUE TITLE
Idea: Create a new cookie_gap list to fill the gaps of cookie warnings

### ISSUE THREAD

**Issue by TheGITofTeo997** [yokoffing/filterlists#7]
Hello,
Since the removal of the **I Don't Care About Cokies** extension, I started to encounter some (not many, but still something) cookie warnings around the websites I visit. 
I decided then, to create a custom filters section in uBlock with these warnings blocked, to fill the gaps of the current lists suggested in this repo.
I started collecting some domains and I thought it would be interesting to have another list here for this gap filling purpose. I can definitely help to keep it updated as soon as i encounter a cookie warning.

Let me know what you think, thanks!

**Comment by yokoffing** [comment]
As was discussed recently in private messages, the unblocked cookie requests is something corrupted in your current build and is pecuilar to you. They are already covered by other lists.

Request denied.
